core/src/test/java/org/apache/iceberg/view/TestViewMetadataParser.java

```diff
@@ -18,18 +18,30 @@
  */
 package org.apache.iceberg.view;
 
+import static org.apache.iceberg.TableMetadataParser.getFileExtension;
```

Comment:
Style: we usually import the defining class and use it to show where static methods came from, rather than using a static import.